Shorten README.md in Python data analysis 3rd workout (#2655)
enkidevs/curriculum: python-data-analysis/python-data-analysis-core/initializing-and-cleaning-datasets/README.md

name: Preparing a Dataset

> description: Learn what Series and DataFrames are, and how to initialize and prepare a dataset for analysis.

Reviewer: Can we abbreviate the description too?

Would something like this work?

```suggestion
description: Learn how to use Series and DataFrames to prepare a dataset for analysis.
```

Author: ```suggestion
description: Learn what Series and Dataframes are, and how to prepare a dataset for analysis.
```
Maybe like this?

Reviewer: Looks good to me
